nilearn/surface/utils.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities for surface mesh, data, images."""
+
+import numpy as np
+
+
+def assert_polydata_equal(data_1, data_2):
+    """Check that 2 PolyData data are equal."""
+    set_1 = set(data_1.parts.keys())
+    set_2 = set(data_2.parts.keys())
+    if set_1 != set_2:
+        diff = set_1.symmetric_difference(set_2)
+        raise ValueError(
+            f"PolyData do not have the same keys. Offending keys: {diff}"
+        )
+
+    for key in data_1.parts:
+        if not np.array_equal(data_1.parts[key], data_2.parts[key]):
+            raise ValueError(
+                f"Part '{key}' of PolyData instances are not equal."
+            )
+
+
+def assert_polymesh_equal(mesh_1, mesh_2):
+    """Check that 2 PolyMeshes are equal."""
+    set_1 = set(mesh_1.parts.keys())
+    set_2 = set(mesh_2.parts.keys())
```

Comment:
Why not create method `equals(data)` in class `PolyMesh` that compares the specified `data` with the instance itself ?